tests/MetaData.test.js

```diff
@@ -0,0 +1,9 @@
+const MetaData = require('../src/MetaData');
+
+test ('Steps down crossing octaves', () => {
```

Comment:
The name of the test should probably be something like 'Constructing the MetaData' or such